README.rst

```diff
@@ -1 +1,152 @@
-Stub readme.
+edx_event_bus_kafka
+=============================
+
+|pypi-badge| |ci-badge| |codecov-badge| |doc-badge| |pyversions-badge|
+|license-badge|
+
+The ``README.rst`` file should start with a brief description of the repository,
+which sets it in the context of other repositories under the ``edx``
+organization. It should make clear where this fits in to the overall edX
+codebase.
+
+Kafka implementation for Open edX event bus.
+
+Overview (please modify)
+------------------------
+This package implements an event bus for Open EdX using Kafka.
+
+The event bus acts as a broker between services publishing events and other services that consume these events. 
+Implementing the event bus will allow for asynchronous messaging across services which greatly improves efficiency as we don't have to wait for scheduled batch synchronizations.
```

Comment:
We should add something about how this will increase reliability as services will not rely on each other to be available at all times.